### PR TITLE
check fastlane column in project and notify scrummaster

### PR DESCRIPTION
**Pull request by individual-it**
this pings the scrummasters every hour if there are items in the fastlane

its a first proposal, more smartness can be added later, specially error checking

### CODE REVIEW

**Review by phil-davis — COMMENTED** [review]
Code looks good.

What do we do about out-of-office times. If an issue is put in the Fastlane at 7pm, this will ping every hour all night. Maybe we let it do that for now, and see how often, and how annoying, that is.

**Review by phil-davis — APPROVED** [review]

File: fastlane.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+"use strict"
+const githubUsername = process.env.HUBOT_GITHUB_USERNAME
+const githubToken = process.env.HUBOT_GITHUB_TOKEN
+const organisation = 'owncloud'
+const projectName = 'Current: QA/CI/TestAutomation'
+const fastLaneColumnName = 'Fastlane'
+const intervalInS = 3600
```
Comment (dpakach) [inline fastlane.js:7]: this interval is too short IMO. I don't think we need to check this often. once every 5 minutes or 10 minutes might be probably ok.

**Review by dpakach — COMMENTED** [review]
Looks good.

**Review by sakshamgurung — COMMENTED** [review]
good

File: fastlane.js (hunk 1 above)
Comment (phil-davis) [reply fastlane.js:7]: IMO it is 3600 seconds = 1 hour @dpakach

**Pull request merged**